docs/readme/examples_source/extractor/train_2loaders_val.md

```diff
@@ -0,0 +1,59 @@
+<details>
+<summary>Training + 2 loaders validation</summary>
+<p>
+
+[comment]:lightning-2loaders-start
+```python
+import pytorch_lightning as pl
+import torch
+
+from oml.datasets.base import DatasetQueryGallery
+from oml.losses.triplet import TripletLossWithMiner, AllTripletsMiner
+from oml.lightning.callbacks.metric import MetricValCallback
+from oml.lightning.modules.extractor import ExtractorModule
+from oml.metrics.embeddings import EmbeddingMetrics
+from oml.models import ViTExtractor
+from oml.transforms.images.torchvision import get_normalisation_resize_torch
+from oml.utils.download_mock_dataset import download_mock_dataset
+
+dataset_root = "mock_dataset/"
+df_train, df_val = download_mock_dataset(dataset_root)
+
+# model
+extractor = ViTExtractor("vits16_dino", arch="vits16", normalise_features=False)
+
+optimizer = torch.optim.SGD(extractor.parameters(), lr=1e-6)
+criterion = TripletLossWithMiner(margin=0.1, miner=AllTripletsMiner())
+
+
+# 1st validation dataset
+val_dataset_1 = DatasetQueryGallery(df_val, dataset_root=dataset_root,
+                                  transform=get_normalisation_resize_torch(im_size=224))
+val_loader = torch.utils.data.DataLoader(val_dataset_1, batch_size=4)
+metric_callback = MetricValCallback(metric=EmbeddingMetrics(extra_keys=[val_dataset_1.paths_key, ],
```

Comment:
you can remove empty arguments for the metric (`[]`), I just did it for debugging